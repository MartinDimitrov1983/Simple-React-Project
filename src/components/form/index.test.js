import React from 'react'
import { render } from '@testing-library/react'

import Form from './index'

test('Snapshot Form', () => {
    const { asFragment } = render(<Form />)
    expect(asFragment()).toMatchSnapshot()
})
