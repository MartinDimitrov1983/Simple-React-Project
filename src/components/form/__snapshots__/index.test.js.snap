// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Form 1`] = `
<DocumentFragment>
  <form
    class="form"
  >
    <p
      class="title"
    >
      Edit properties
    </p>
    <div
      class="container"
    >
      <label
        class="label"
        for="name"
      >
        Name:
      </label>
      <input
        class="formControl"
        id="name"
        name="name"
        placeholder="Name"
        required=""
        type="text"
        value=""
      />
    </div>
    <div
      class="container"
    >
      <label
        class="label"
        for="plan"
      >
        Plan Number:
      </label>
      <input
        class="formControl"
        id="plan"
        name="plan"
        placeholder="Plan Number"
        required=""
        type="text"
        value=""
      />
    </div>
    <div
      class="container"
    >
      <label
        class="label"
        for="units"
      >
        Units:
      </label>
      <input
        class="formControl"
        id="units"
        name="units"
        placeholder="Units"
        required=""
        type="number"
        value=""
      />
    </div>
    <div
      class="container"
    >
      <label
        class="label"
        for="city"
      >
        City:
      </label>
      <input
        class="formControl"
        id="city"
        name="city"
        placeholder="City"
        required=""
        type="text"
        value=""
      />
    </div>
    <div
      class="container"
    >
      <label
        class="label"
        for="region"
      >
        Region:
      </label>
      <input
        class="formControl"
        id="region"
        name="region"
        placeholder="Region"
        required=""
        type="number"
        value=""
      />
    </div>
    <div
      class="container"
    >
      <label
        class="label"
        for="manager"
      >
        Manager:
      </label>
      <input
        class="formControl"
        id="manager"
        name="manager"
        placeholder="Manager"
        required=""
        type="number"
        value=""
      />
    </div>
    <div
      class="container"
    >
      <label
        class="label"
        for="previous-manager"
      >
        Previous Manage:
      </label>
      <input
        class="formControl"
        id="previous-manager"
        name="previousManager"
        placeholder="Previous Manager"
        required=""
        type="number"
        value=""
      />
    </div>
    <div
      class="container"
    >
      <label
        class="label"
        for="management-company"
      >
        Management Company:
      </label>
      <input
        class="formControl"
        id="management-company"
        name="managementCompany"
        placeholder="Management Company"
        required=""
        type="text"
        value=""
      />
    </div>
    <div
      class="container"
    >
      <label
        class="label"
        for="plan-registered"
      >
        Plan Registered:
      </label>
      <input
        class="formControl"
        id="plan-registered"
        name="planRegistered"
        placeholder="Plan Registered"
        required=""
        type="date"
        value=""
      />
    </div>
    <div
      class="container"
    >
      <label
        class="label"
        for="address"
      >
        Address:
      </label>
      <input
        class="formControl"
        id="address"
        name="address"
        placeholder="Address"
        required=""
        type="text"
        value=""
      />
    </div>
    <div
      class="container"
    >
      <label
        class="label"
        for="account"
      >
        Account:
      </label>
      <input
        class="formControl"
        id="account"
        name="account"
        placeholder="Account"
        required=""
        type="text"
        value=""
      />
    </div>
    <div
      class="container"
    >
      <label
        class="label"
        for="abn"
      >
        abn:
      </label>
      <input
        class="formControl"
        id="abn"
        name="abn"
        placeholder="abn"
        required=""
        type="text"
        value=""
      />
    </div>
    <button
      class="btn"
      type="submit"
    >
      Save
    </button>
  </form>
</DocumentFragment>
`;
